packages/kbn-cases-components/tsconfig.json

```diff
@@ -14,6 +14,7 @@
   ],
   "kbn_references": [
     "@kbn/i18n",
+    "@kbn/i18n-react",
```

Comment:
this was added by vs code. I had to use `@kbn/i18n-react` because module '"@kbn/i18n"' has no exported member 'FormattedRelative'.